src/ScheduleFunctions.cpp

```diff
@@ -1368,20 +1222,20 @@ class InjectGroupRealization : public IRMutator2 {
             const auto &f2_it = env.find(pair.func_2);
             internal_assert(f2_it != env.end());
             const vector<Dim> &dims_2 =
-                (pair.stage_2 == 0) ? f2_it->second.definition().schedule().dims() :
-                                      f2_it->second.update(pair.stage_2-1).schedule().dims();
+                    (pair.stage_2 == 0) ? f2_it->second.definition().schedule().dims() :
+                    f2_it->second.update(static_cast<int>(pair.stage_2 - 1)).schedule().dims();
 
             const auto &iter = std::find_if(dims.begin(), dims.end(),
-                [&pair](const Dim &d) { return var_name_match(d.var, pair.var_name); });
+                                            [&pair](const Dim &d) { return var_name_match(d.var, pair.var_name); });
             internal_assert(iter != dims.end());
-            start_fuse = std::min(start_fuse, (size_t)(iter - dims.begin()));
+            start_fuse = std::min(start_fuse, (size_t) (iter - dims.begin()));
```

Comment:
No space after C-style cast. We have a clang-format file that I think would fix this.

Reply:
Is that universally true? If that's preferred, I'm not sure we're enforcing it well. (NBD to be personally either way.)